deposit-app/src/test/java/edu/unc/lib/boxc/deposit/fcrepo4/IngestContentObjectsJobIT.java

```diff
@@ -404,6 +401,48 @@ public void ingestSimpleFile() throws Exception {
         assertLinksToDepositRecord(primaryObj);
     }
 
+    @Test
+    public void ingestWorkWithStreamingFileObject() throws Exception {
+        String label = "testwork";
+        PID workPid = pidMinter.mintContentPid();
+
+        // Construct the deposit model with work object
+        Model model = job.getWritableModel();
+        Bag depBag = model.createBag(depositPid.getRepositoryPath());
+
+        // Constructing the work in the deposit model with a label
+        Bag workBag = model.createBag(workPid.getRepositoryPath());
+        workBag.addProperty(RDF.type, Cdr.Work);
+        workBag.addProperty(CdrDeposit.label, label);
+        depBag.add(workBag);
+
+        var filePid = addStreamingFileObject(workBag);
+
+        job.closeModel();
+
+        job.run();
+
+        treeIndexer.indexAll(baseAddress);
+
+        ContentContainerObject destObj = (ContentContainerObject) repoObjLoader.getRepositoryObject(destinationPid);
+        List<ContentObject> destMembers = destObj.getMembers();
+        assertEquals(1, destMembers.size(), "Incorrect number of children at destination");
+
+        // Make sure that the work is present and is actually a work
+        WorkObject mWork = (WorkObject) findContentObjectByPid(destMembers, workPid);
+
+        String title = mWork.getResource().getProperty(DC.title).getString();
+        assertEquals(label, title, "Work title was not correctly set");
+        assertClickCount(2);
+        ingestedObjectsCount(2);
+
+        // Check the right number of members are present
+        List<ContentObject> workMembers = mWork.getMembers();
+        assertEquals(1, workMembers.size(), "Incorrect number of members in work");
+        FileObject fileObj = (FileObject) findContentObjectByPid(workMembers, filePid);
+        assertNotNull(fileObj);
```

Comment:
we should also assert that the fileobject has the streaming properties with the correct values. We should be able to find them by checking fileObj.getResource()